### PR TITLE
Check if debug log is enabled before record a log

### PR DESCRIPTION
**Pull request by cezarlz**

### CODE REVIEW

**Comment by cezarlz** [comment]
This isn't a hotfix... It's a new version but we opened a exception to upload the changes today.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: php
 
 env:
-  EBANX_WC_PLUGIN_VERSION: 3.0.6
+  EBANX_WC_PLUGIN_VERSION: 3.0.7
```
Comment (SparK-Cruz) [inline .travis.yml]: Can we rename to EBANX_WC_VERSION? The Plugin word there is really confusing.

Comment (gpressutto5) [inline .travis.yml]: I think it would be better to rename to WC_VERSION since only our own docker container will read it.

Comment (cezarlz) [inline .travis.yml]: Sure!

**Pull request merged**